sdk/services/synths.ts

```diff
@@ -30,8 +30,8 @@ export default class SynthsService {
 			currencyKeys,
 			synthsBalances,
 			synthsUSDBalances,
-		]: SynthBalancesTuple = await this.sdk.contracts.SynthUtil.connect(
-			this.sdk.provider
+		]: SynthBalancesTuple = await this.sdk.context.contracts.SynthUtil.connect(
```

Comment:
Is this connected on context so wouldn't need to connect here?

Reply:
Good catch. I missed this one from when I made the changes across the SDK initially.